be/src/runtime/tablets_channel.h

```diff
@@ -199,6 +204,7 @@ class TabletsChannel : public RefCountedThreadSafe<TabletsChannel> {
     // next sequence we expect
     std::atomic<int> _num_remaining_senders;
     std::vector<Sender> _senders;
+    size_t _max_sliding_window_size = config::max_load_dop * 3;
```

Comment:
Initialize this field variable in the initialization list, as other fields. Keep code style consistent within the same scope.